recipes/pseudonetcdf/meta.yaml

```diff
@@ -0,0 +1,66 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
@barronh - I think you can safely remove the commented out lines at this point.

Reply:
Thanks, done.